pkg/arvo/app/chat.hoon

```diff
@@ -341,195 +253,188 @@
     (mean [leaf+"invalid wire for diff: {(spud wir)}"]~)
   ::
   ::  %circles
-      %circles
-    ?>  ?=(%circles -.rum)
-      =/  cis
+  ::
+    %circles
+  ?>  ?=(%circles -.rum)
+  =/  cis
+    ?:  add.rum
+      (~(put in circles.str) cir.rum)
+    (~(del in circles.str) cir.rum)
+  =/  sta
+    %=  str
+      circles  cis
+      peers
         ?:  add.rum
-          (~(put in circles.str.sta) cir.rum)
-        (~(del in circles.str.sta) cir.rum)
-      =/  str
-        %=  str.sta
-          circles  cis
-          peers
-            ?:  add.rum
-              (~(put by peers.str.sta) [our.bol cir.rum] ~)
-            (~(del by peers.str.sta) [our.bol cir.rum])
-        ==
-      :-  (send-chat-update [[%circles cis] str])
-      this(str.sta str)
+          (~(put by peers.str) [our.bol cir.rum] ~)
```

Comment:
You want to indent these (long-form `%=` changes) similarly to the `?+` cases, where `peers` would be indented deeper than the code below it. Also probably separate the wide-form from the tall-form with a `::` in the left margin of this block.

Reply:
Also, you generally want `%_` instead of `%=`. It's the same thing, except makes sure the end result still has the same type.

Reply:
Changed this in a bunch of places. Thanks!